packages/flutter_tools/lib/src/commands/assemble.dart

```diff
@@ -148,12 +147,6 @@ class AssembleCommand extends FlutterCommand {
   @override
   String get category => FlutterCommandCategory.project;
 
-  @override
-  Future<CustomDimensions> get usageValues async => CustomDimensions(
```

Comment:
is unified_analytics sending this same data? and if so, is it essentially using a copy past of this logic?

Reply:
Yes:

https://github.com/flutter/flutter/blob/a647a1df3cd4f283c8699e2bfc6d51b305ac6f88/packages/flutter_tools/lib/src/commands/assemble.dart#L150-L156

Reply:
thanks, LGTM